app/src/main/java/app/revanced/manager/ui/screen/settings/AdvancedSettingsScreen.kt

```diff
@@ -164,6 +165,12 @@ fun AdvancedSettingsScreen(
     }
 }
 
+@Preview
+@Composable
+private fun APIUrlDialogPreview() {
+    APIUrlDialog("https://example.com") {}
```

Comment:
Should this not be the default API by default?

Reply:
Any value is fine here since this is only shown to developer using Android Studio or any IDE that support rendering a composable.

Reply:
I guess using default api makes more sense then

Reply:
This section is only to preview the design of the composable, it can be removed from this PR with no consequence

Reply:
I am not talking specifically about the section but the text shown in the section. So then either removing or using the default api here works